common/app/routes/Challenges/redux/index.js

```diff
@@ -237,10 +237,11 @@ export const challengeMetaSelector = createSelector(
     const title = blockName && challenge.title ?
       `${blockName}: ${challenge.title}` :
       challenge.title;
-
+    const guideUrl = challenge.guideUrl;
     return {
       type,
       title,
+      guideUrl,
```

Comment:
If this is already on the challenge object, do we need to add it to the meta information? meta is reserved for derived data.